charts/bandstand-headless-service/templates/scaled-object.yaml

```diff
@@ -36,5 +36,10 @@ spec:
         value: {{ quote (.threshold | default 75) }}
       metricType: Utilization
     {{- end }}
+    {{- if eq .type "kafka" }}
+    - type: kafka
```

Comment:
I obviously missed adding kafka to this chart, it should be identical to the web services chart. Can just copy the whole thing over

Reply:
Thanks